plugins/node/instrumentation-runtime-node/package.json

```diff
@@ -45,7 +45,7 @@
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/sdk-metrics": "^1.20.0",
     "@types/mocha": "^10.0.6",
-    "@types/node": "^20.11.2",
+    "@types/node": "^22.1.0",
```

Comment:
compilation is failing due to a incompatibility between `@node/types` and `typescript`. Max versions we can update without errors is `v18.18.14`
 ref: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2462

note: this will be fixed when TypeScript is updated

Reply:
fix